client/blocks/comment-detail/comment-detail-actions.jsx

```diff
@@ -13,21 +13,33 @@ import { includes } from 'lodash';
 /**
  * Internal dependencies
  */
+import ButtonGroup from 'components/button-group';
 import Button from 'components/button';
 
 const commentActions = {
-	unapproved: [ 'like', 'approve', 'edit', 'spam', 'trash' ],
-	approved: [ 'like', 'approve', 'edit', 'spam', 'trash' ],
+	unapproved: [ 'like', 'approve', 'edit', 'reply', 'spam', 'trash' ],
+	approved: [ 'like', 'approve', 'edit', 'reply', 'spam', 'trash' ],
 	spam: [ 'approve', 'delete' ],
 	trash: [ 'approve', 'spam', 'delete' ],
 };
 
 const hasAction = ( status, action ) => includes( commentActions[ status ], action );
 
+const CommentDetailActionsContainer = ( { compact, children } ) =>
+	compact ? (
+		<div className="comment-detail__actions is-quick">
+			<ButtonGroup>{ children }</ButtonGroup>
+		</div>
+	) : (
+		<div className="comment-detail__actions">{ children }</div>
+	);
```

Comment:
Minor point, but can we clean this up later to stick with one component per file? Another option would be to have CommentDetailActions extend Component, to give us more flexibility with helper methods.